sotodlib/io/load_smurf.py

```diff
@@ -1134,14 +1234,36 @@ def index_metadata(
             max_ctime=max_ctime,
             stop_at_error=stop_at_error,
         )
+        
+        session.close()
+
+    def index_action_observations(
+        self, min_ctime=16000 * 1e5, max_ctime=None, stop_at_error=False
+    ):
```

Comment:
Not sure if I understand this one, why create a separate function that wraps `index_observations`? Is this only so that you don't need to pass in a session object?